src/pages/reactivistas/index.tsx

```diff
@@ -17,13 +20,49 @@ const ReactGroupPage: React.FC = () => {
                 </h1>
                 <h2 className="font-medium leading-7 text-md text-primary sm:leading-9 sm:truncate">
                   Propuesta de nuevo grupo de estudio
+                  <span className="text-black mx-2">-</span>
+                  <span
+                    className="cursor-pointer text-blue-400"
+                    onClick={() => setIsOpen(true)}
+                  >
+                    Ver Requisitos
+                  </span>
                 </h2>
               </div>
             </div>
           </div>
           <ReactGroupForm />
         </div>
       </div>
+      <button></button>
+
+      <Modal isOpen={isOpen} close={() => setIsOpen(false)} title="Leer">
```

Comment:
Can we put `Requisitos para iniciar un nuevo grupo` as the title and then leave it out of the content part?